src/Microsoft.OData.ModelBuilder/Conventions/Attributes/NullableAttributeExtensions.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.OData.ModelBuilder.Conventions.Attributes
```

Comment:
>Conventions.Attributes [](start = 39, length = 22)

If it's for "PropertyConfiguration", it might be in "Property" folder?

Reply:
Fied. Renamed to PropertyConfiguration and moved to Property folder.